concise-mid-tag.cddl

```diff
@@ -1,136 +1,116 @@
-; Notes:
-; - linked-tags contains references to dependent CoSWID and
-;   CoMID tags
-; - claims may be omitted for manifests that only capture
-;   dependencies to other manifests
 concise-mid-tag = {
   ? comid.language => language-type
   comid.tag-identity => tag-identity-map
-  ? comid.module-name => element-name-map
-  ? comid.entity => one-or-more<module-entity-map>
+  ? comid.entity => one-or-more<entity-map>
   ? comid.linked-tags => one-or-more<linked-tag-map>
-  ? comid.claims => claims-map
+  comid.triples => triples-map
   * $$concise-mid-tag-extension
 }
 
 language-type = text
 
 tag-identity-map = {
   comid.tag-id => $tag-id-type-choice
-  comid.tag-version => tag-version-type
-  * $$tag-identity-map-extension
+  ? comid.tag-version => tag-version-type
 }
 
 $tag-id-type-choice /= tstr
 $tag-id-type-choice /= uuid-type
 
 tag-version-type = uint .default 0
 
-module-entity-map = {
+entity-map = {
```

Comment:
Please remove module word from role-type-choice also...

Reply:
mmm why?  we use a similar convention for corim and xcorim roles.  Maybe we could change the prefix from `module` to `comid`, but I don't understand why we should remove it altogether

Reply:
changed `module` into `comid` e77668c

Reply:
Works for me!